app/pages/cif/project-revision/[projectRevision]/create.tsx

```diff
@@ -89,21 +94,25 @@ export function ProjectRevisionCreate({
     <>
       <DefaultLayout session={session} leftSideNav={taskList}>
         <div>
-          <FormBase
-            id="ProjectRevisionCreateForm"
-            schema={localSchema as JSONSchema7}
-            uiSchema={projectRevisionUISchema}
-            onSubmit={handleCreateRevision}
-            ObjectFieldTemplate={EmptyObjectFieldTemplate}
-          >
-            <Button
-              type="submit"
-              size="small"
-              disabled={isCreatingProjectRevision}
+          {existingRevision ? (
+            <h1>There is an existing revision</h1>
```

Comment:
I thought we would redirect the user? we can leave this for now, but we need to confirm with @nanyangpro what the behaviour should be.

Reply:
yup the other card still will need to address this.